From baf8a53af103424248e9f245b15766da8f1a2f7a Mon Sep 17 00:00:00 2001
From: Kathiravan Thirumoorthy <quic_kathirav@quicinc.com>
Date: Fri, 15 Dec 2023 14:53:14 +0530
Subject: [PATCH 438/500] ARM: mach-qcom: Add support for DEVSOC

DEVSOC has GICV3 IRQ controller. Enable the GICV3 driver under the
DEVSOC config, since other targets doesn't need this support.

Change-Id: I45b7509b55a4e41c9cb722841c37bfaada977571
Signed-off-by: Kathiravan Thirumoorthy <quic_kathirav@quicinc.com>
---
 arch/arm/mach-qcom/Kconfig | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm/mach-qcom/Kconfig b/arch/arm/mach-qcom/Kconfig
index 6a8be97e905a..f637d63e2e37 100644
--- a/arch/arm/mach-qcom/Kconfig
+++ b/arch/arm/mach-qcom/Kconfig
@@ -17,6 +17,10 @@ config ARCH_IPQ40XX
 	select CLKSRC_QCOM
 	select HAVE_ARM_ARCH_TIMER
 
+config ARCH_DEVSOC
+	bool "Enable support for DEVSOC"
+	select ARM_GIC_V3
+
 config ARCH_MSM8X60
 	bool "Enable support for MSM8X60"
 	select CLKSRC_QCOM
-- 
2.34.1

